stdlib/ballerina-http/src/main/ballerina/http/http_headers.bal

```diff
@@ -16,53 +16,71 @@
 
 
 // Please maintain alphabetical order when adding new header names
-documentation { HTTP header key 'age'. Gives the current age of a cached HTTP response. }
+documentation { HTTP header key `age`. Gives the current age of a cached HTTP response. }
 @final public string AGE = "age";
 
-documentation { HTTP header key 'authorization' }
+documentation { HTTP header key `authorization` }
 @final public string AUTHORIZATION = "authorization";
 
-documentation { HTTP header key 'cache-control'. Specifies the cache control directives required for the function of HTTP caches. }
+documentation {
+    HTTP header key `cache-control`. Specifies the cache control directives required for the function of HTTP caches.
+}
 @final public string CACHE_CONTROL = "cache-control";
 
-documentation { HTTP header key 'content-length'. Specifies the size of the response body in bytes. }
+documentation { HTTP header key `content-length`. Specifies the size of the response body in bytes. }
 @final public string CONTENT_LENGTH = "content-length";
 
-documentation { HTTP header key 'date'. The timestamp at the time the response was generated/received. }
+documentation { HTTP header key `content-type`. Specifies the type of the message payload. }
+@final public string CONTENT_TYPE = "content-type";
+
```

Comment:
We already have constants for content-type, content-length in mime and since mime is a dependency in http would it really necassary have these here in http as well?

Reply:
I guess there would be some overlap between MIME headers and HTTP ones, like in this case. For HTTP use cases we'll use the ones in the HTTP package.